### PR TITLE
EN-5464 renamed vid cookie from vid/_pxvid to pxvid

### PR DESCRIPTION
**Pull request by pxjohnny**

### CODE REVIEW

File: lib/px/utils/pxclient.lua [hunk 1]
```diff
@@ -242,6 +242,8 @@ function M.load(config_file)
 
         if ngx.var.cookie__pxvid then
             vid = ngx.var.cookie__pxvid
+        elseif ngx.var.cookie_pxvid then
+            vid = ngx.var.cookie_pxvid
         elseif ngx.var.cookie_vid then
```
Comment (yaronpx) [inline lib/px/utils/pxclient.lua]: this is not needed anymore, if I understand correctly.
@dimapx ?

Comment (pxjohnny) [reply lib/px/utils/pxclient.lua]: @dimapx  said when i checked with him on slack that  we need to check for both _pxvid and pxvid (nodejs does the same)

Comment (yaronpx) [reply lib/px/utils/pxclient.lua]: but not `vid`...

Comment (dimapx) [reply lib/px/utils/pxclient.lua]: @pxjohnny @yaronpx `pxvid` replaces `vid`, such that the check should be against `_pxvid` and `pxvid`. no need to check for a `vid` cookie key. 
In node it is exactly as it should be - https://github.com/PerimeterX/perimeterx-node-core/blob/master/lib/pxproxy.js#L108.

**Pull request merged**